loot_importer.user.js

```diff
@@ -126,50 +156,70 @@ function ParseLootPage() {
+  if (releaseartist == "Various Artists") {
+    // Everything is: title(format)
+    releaseartisttitle_regex = /(.*)\((.*)\)/;
+    releaseartisttitle = AlbumName[0].innerText.match(releaseartisttitle_regex);
+
+
+    releasetitle = releaseartisttitle[1];
+    release_format = releaseartisttitle[2];
+
+  } else {
+    // 
+    releaseartisttitle_regex = /(.*) - (.*)\((.*)\)/;
+    releaseartisttitle = AlbumName[0].innerText.match(releaseartisttitle_regex);
+
+    releasetitle = releaseartisttitle[2].trim();
+    releaseartist = releaseartisttitle[1];
+    release_format = releaseartisttitle[3];
 
-  // select the product info
-  var ReleaseInfo = document.querySelectorAll("div#tab-1 > table > tbody > tr > *");
-  LOGGER.debug("Product Info: ", ReleaseInfo);
-
-  for (var prodinfoloop = 0; prodinfoloop < ReleaseInfo.length; prodinfoloop++) {
-    var prodinfolabel = ReleaseInfo[prodinfoloop];
-    //LOGGER.debug("** ProdInfo LABEL: ",prodinfolabel);
-    if (prodinfolabel.tagName == "TD") {
-      var prodinfolabellowcase = prodinfolabel.innerText.toLowerCase().trim(); //replaced textContent with innerText
-
-      LOGGER.debug("<TD>", prodinfolabellowcase);
-
-      switch (prodinfolabellowcase) {
-        case "label:": // use these cases to select the spesific text values
-          LOGGER.debug("* label: selected *");
-          var release_label = new Object();
-          release_label.name = prodinfolabel.nextElementSibling.innerText.trim();
-          prodlabels.push(release_label);
-          break;
-        case "release date:": // "release date:" TODO: NOT WORKING
-          LOGGER.debug("* release date: *");
-          var date_regex = /([A-Z][a-z]+)\s(\d{4})/; // October 2014 [1] [2]
-          // need to add month and year code
-          break;
-        case "performers:": // use these cases to select the spesific text values
-          LOGGER.debug("* performers: selected *");
-          var release_artist_credit_object = new Object();
-          release_artist_credit_object.artist_name = prodinfolabel.nextElementSibling.innerText.trim();
-          release_artist_array.push(release_artist_credit_object);
-          break;
-        case "format:":
-          LOGGER.debug("* format: selected *");
-          release_format = prodinfolabel.nextElementSibling.childNodes[0].alt;
-          LOGGER.debug("** format: value **", release_format);
-          break;
-      }
-    }
   }
 
+
+
+  LOGGER.debug("Release Title:", releasetitle, "  Release Artist:", releaseartist, "  Release Format:", release_format);
+
+
+  // extract all tr from table with class productDetails
+  $("table.productDetails tr").each(function() {
+    // get text from first td, trim and convert it to lowercase
+    var prodinfolabellowcase = $(this).children('td').eq(0).text().trim().toLowerCase();
+    prodinfolabellowcase = prodinfolabellowcase.replace(/\s+/g, ''); //removing white spaces as switch isnt matching spaces for some reason
```

Comment:
Huh? `switch()` should have no issue matching strings with spaces...
However, i think you can simply use `if` / `else if` here, it will not make much difference.